packages/react-widgets/src/widgets/utils/sortLayersById.js

```diff
@@ -0,0 +1,16 @@
+import { assert } from '@carto/react-core';
+
+/**
+ * Sorts a list of layers based on a list of id property values
+ * @param {any[]} layers list of layers to sort
```

Comment:
```suggestion
 * @param {{ id: string }[]} layers - array of layers to sort
```

Btw, usually between the type and the description we put a slash